### PR TITLE
docs: "getting involved" page

### PR DESCRIPTION
**Pull request by wodeni**
# Description

This PR adds a "getting involved" page on the website to encourage visitors to join the Penrose community and guide them to action items. The content on the page is by no means final. This PR mainly focuses on adding the infra for content in future PRs.

# Implementation strategy and design decisions

* To de-clutter the nav bar, I removed the news and join discord buttons. The content under news are mostly papers, which we should link to in the getting involved page as examples of research outcome. The discord link is already associated with the top-right icon.
* Add github icons of contributors to the team page
* Add a roadmap section and components for linking to github issues and prs. The content is TBD 

# Examples with steps to reproduce them

See deployment

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have reviewed any generated registry diagram changes

### CODE REVIEW

**Comment by kyleliangus** [comment]
Can't wait to get started!

File: packages/docs-site/community.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  /* .research {
+    grid-row: span 1
+  }
+  .code-and-docs {
+    grid-row: span 1
+  } */
+</style>
+
+[discord]: https://discord.gg/a7VXJU4dfR
+[email]: mailto:team@penrose.ink
```
Comment (samestep) [inline packages/docs-site/community.md:123]: Is this email address up to date?

File: packages/docs-site/community.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+- Automatic differentiation based on [Rose][rose] 🌹 <PR pr="1636" />
+- Layout optimization on WebWorkers <PR pr="1528" />
+
+### 🎯 Planned
+
+- Next iteration on [SolidJS API for Penrose](/docs/ref/solid.md) and docs
+- Implement high-quality hidden strokes <Issue issue="517" />
+- Support arbitrary Penrose trios in [`obsidian-penrose-plugin`]
+- A single file format (`.penrose`) for Penrose trios
```
Comment (samestep) [inline packages/docs-site/community.md:86]:
```suggestion
- A single file format (`.penrose`) for Penrose diagrams
```

**Review by samestep — APPROVED** [review]
Looks great! Just had a couple minor comments; I'm guessing others will have more substantial feedback.

File: packages/docs-site/community.md (hunk 1 above)
Comment (wodeni) [reply packages/docs-site/community.md:123]: good question. looks like the address is down. need to set it up again

**Pull request merged**